dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,131 @@
+| get     | `dmctl task get --output="new_task.yaml" task1`                                                                                     | --output(default: ".")                                                                                                  | task-name              | show the task config in yaml format, also support output to file. |
+| list    | `dmctl task list --stage="Running" --source="source1,source2"`                                                                      | --source(default: ""), --stage(default: "")                                                                             |                        | list all tasks in current cluster.                                |
+| status  | `dmctl task status --source="source1,source2" task1`                                                                                | --source(default: "")                                                                                                   | task-name              | show task detail status.                                          |
+| start   | `dmctl task start --source="source1,source2" --remove-meta --start-time="2021-01-01 00:00:00" --safe-mode-time-duration="1s" task1` | --source(default: ""), --remove-meta(default: false), --start-time(default: ""), --safe-mode-time-duration(default: "") | task-name              | start a stopped task with many flags.                             |
+| stop    | `dmctl task stop --source="source1,source2" --timeout="60s" task1`                                                                  | --source(default: ""), --timeout(default: "10s")                                                                        | task-name              | stop a running task with many flags.                              |
+
+### dmctl commands for Source
+
+| Command  | Full Syntax Example                                | Flags                   | Arguments                | Description                                                         |
+|----------|----------------------------------------------------|-------------------------|--------------------------|---------------------------------------------------------------------|
+| create   | `dmctl source create source1.yaml`                 |                         | config-file              | create source with config file.                                     |
+| update   | `dmctl source update source1 source1.yaml`         |                         | source-name, config-file | update a source with config file.                                   |
+| delete   | `dmctl source delete --force source1`              | --force(default: false) | source-name              | delete a source with config file.                                   |
+| get      | `dmctl source get --output="source1.yaml" source1` | --output(default: ".")  | source-name              | show the source config in yaml format, also support output to file. |
```

Comment:
default `.` means we always write a file to current directory, but what's the filename?

Also it not "also support output to file" because there's no alternative to output to file

Reply:
update some default value to `can be omitted`

>  because there's no alternative to output to file

it can be printed on screen

Reply:
“optional” means "can be omitted", no need to repeat it.